phpunit.xml

```diff
@@ -23,6 +23,9 @@
         <testsuite name="Links">
             <directory suffix="Test.php">./domains/Links/Tests</directory>
         </testsuite>
+        <testsuite name="Accounts">
```

Comment:
Let's keep this test suite structure ordered alphabetically. They'll tend to increase, in number, and not having some natural order will hurt navigability.

Reply:
ascending or descending?
...just kidding. Done